src/chartmuseum/Chart.yaml

```diff
@@ -15,4 +15,4 @@ sources:
   - https://github.com/helm/chartmuseum
 maintainers:
   - name: chartmuseum
-    url: https://github.com/chartmuseum
+    url: https://github.com/chartmuseum
```

Comment:
```suggestion
    url: https://github.com/chartmuseum
   
```

Reply:
hmm i will review this in more detail in a few minutes

Reply:
thanks @fede-r1c0, we'll just need to make sure there is a trailing new line at the end of the file and no trailing spaces

Reply:
done! i fixed it